### PR TITLE
Add extra meter sensors (EG, Eastron meter)

### PR DESCRIPTION
**Pull request by special-K-Ken**
Add a couple of extra sensors to measure grid under current and voltage - ie the Eastron or similar meter (not EPM).

This is from the Solis spec and I've verified the data is returned in another app (c#).

### CODE REVIEW

File: custom_components/solis/soliscloud_api.py [hunk 1]
```diff
@@ -126,6 +126,8 @@
         SOC_DISCHARGE_SET:                ['socDischargeSet', float, 0],
         BYPASS_LOAD_POWER:                ['bypassLoadPower', float, 3],
         BYPASS_LOAD_POWER_STR:            ['bypassLoadPowerStr', str, None],
+        METER_ITEM_A_CURRENT:             ['meterItemACurrent', float, 3],
```
Comment (hultenvp) [inline custom_components/solis/soliscloud_api.py:129]: How can this work? According to the 2.0 spec the fields are called 'iA' and 'uA' respectively. Did you test your changes with HomeAssistant?

**Review by hultenvp — CHANGES_REQUESTED** [review]
Please test with HA. I cannot modify and test myself

File: custom_components/solis/soliscloud_api.py (hunk 1 above)
Comment (special-K-Ken) [reply custom_components/solis/soliscloud_api.py]: You're right, I have iA and uA in my code but pasted the wrong fields in here. Thanks

**Comment by special-K-Ken** [comment]
Extra fields added, and corrected the fields from inverterDetail API

**Comment by special-K-Ken** [comment]
image

No errors reported in mass terminal

**Review by hultenvp — APPROVED** [review]

**Pull request merged**